internal/service/alarms/internal/dictionary/dictionary.go

```diff
@@ -21,15 +21,18 @@ const (
 )
 
 // TODO: create a file with the resource type names once they are defined by the Resource Server
-const resourceTypeCluster = "Cluster"
+const (
+	resourceTypeCluster = "Cluster"
+	resourceTypeHub     = "Hub"
+)
 
 type AlarmDictionary struct {
-	Client client.Client
+	Client cr_client.Client
 
 	RulesMap map[int][]monitoringv1.Rule
 }
 
-func New(client client.Client) *AlarmDictionary {
+func New(client cr_client.Client) *AlarmDictionary {
```

Comment:
I think we should send back these errors to the init code? If anything of this fails we shouldnt start the server

Reply:
No, I don't think so. For some reason the collection of rules for a resource type can fail but that doesn't mean we need to stop. The server can still work. DB can have data already. What I think is we should implement a resync mechanism so that I tries to populate the dict again.

Reply:
There might be a resource type for which there are no managed clusters. Maybe the resource server is not in sync. An error will be returned but we can still continue. I think this section is not critical for the alarm server to continue or not. In my opinion, we should simply log and continue.

This is outside the scope of this PR. We need too continue this discussion for sure

Reply:
> An error will be returned but we can still continue

Yeah this is not true for all error right? yeah sounds let's move to another pr as needed.